File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Control Flow Structures/Python/Excercises/program-7.py
# 7-Escribir un programa que pida al usuario un número entero y muestre por pantalla si es múltiplo de 45.


userInput = int(input('Ingresa un numero entero: '))

if userInput % 45 == 0:
    print(f'{True}, El numero {userInput} es multiplo de 45!')
else:
    print(f'{False}, El numero {userInput} no es multiplo de 45!')

File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Control Flow Structures/Javascript/Excercises/program-7.js

// 7 - Escribir un programa que pida al usuario un número entero y muestre por pantalla si es múltiplo de 45.


let userInput = parseInt(prompt('Ingresa un numero entero: '))

if (userInput % 45 == 0) {
    console.log(`${true}, El numero ${userInput} es multiplo de 45!`)
} else {
    console.log(`${false}, El numero ${userInput} no es multiplo de 45!`)
}